web.primefaces/licenseinfo.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/web/primefaces/resources/welcomePrimefaces.template</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/jsfcrud.css</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/jsfcrud.js</file> 
+        <license ref="Apache-2.0-ASF" />
+        <comment type="TEMPLATE_MINIMAL_IP" />
+    </fileset>
+    <fileset>
+        <file>src/org/netbeans/modules/web/primefaces/snippets/edit.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/snippets/table.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/snippets/view.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/JsfUtil.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/bundle.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/controller.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/create.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/edit.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/list.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/template.ftl</file>
+        <file>src/org/netbeans/modules/web/primefaces/templates/view.ftl</file>
+        <license ref="CDDL-1.0" />
+        <comment type="CATEGORY_B" />
```

Comment:
These files are part of the donation, so please update the license to ALv2, then they don't need comment and are just a regular part of the repository.